examples/grids/Pipfile

```diff
@@ -0,0 +1,12 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+automata = {git = "https://github.com/automata-tech/eva_python_sdk.git",ref = "development"}
```

Comment:
Shouldn't you pin it to the actual version of the SDK, otherwise the examples might stop working when development gets updated.

Reply:
> TODO before merge: change the Pipfile to use SDK version 2.x.x
I'm waiting for james' teach lock fixes so I'll need to change this before merge to our sexy new pypi dependancy